modules/overrides/overrides.go

```diff
@@ -12,11 +12,20 @@ import (
 	"github.com/grafana/dskit/runtimeconfig"
 	"github.com/grafana/dskit/services"
 	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
 	"gopkg.in/yaml.v2"
 )
 
 const wildcardTenant = "*"
 
+var (
+	metricOverridesLimits = promauto.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: "tempo",
+		Name:      "limits",
```

Comment:
bikeshedding on naming but should we name this `overrides` so it's consistent with cortex?
https://github.com/cortexproject/cortex/blob/c7845aedc3f045a83aa9c27d029391fc4b244d30/pkg/util/validation/exporter.go#L19

Reply:
Can do.